apps/essence/server/services/db/repository/media.ts

```
@@ -0,0 +1,53 @@
+      .update(mediaTable)
+      .set(data)
+      .where(eq(mediaTable.id, id))
+      .returning()
+
+    return media
+  }
+
+  static async delete(id: string) {
+    return useDatabase().delete(mediaTable).where(eq(mediaTable.id, id))
+  }
+}
```

Biome (1.9.4)

[error] Avoid classes that contain only static members.

Prefer using simple functions instead of classes with only static members.

(lint/complexity/noStaticOnlyClass)